tests/linearizability/identity/lease_ids.go

```diff
@@ -15,39 +15,84 @@
 package identity
 
 import (
+	"math/rand"
 	"sync"
 )
 
 type LeaseIdStorage interface {
 	LeaseId(int) int64
 	AddLeaseId(int, int64)
 	RemoveLeaseId(int)
+	GetRandomClientWithLease() (int, int64)
+}
```

Comment:
How I imagine the interface for leases.
```suggestion
type LeaseIdStorage interface {
    // Add adds lease to storage. Should be called after LeaseGrant. Increases number leases.
	Add(leaseId int64)
	// Remove permanently removes lease from storage. Should be called after LeaseRevoke. Decreases number of leases.
    Remove(leaseId int64)
    // Take returns a no in use lease from storage and marks it as in use. Should be called before Put with Lease. Doesn't change number of leases. Can return 0 if there are no leases, or all leases are in use.
    Take() leaseId
    // Return returns in use lease to storage so it can be used later. Should be called after Put with Lease. Doesn't change number of leases. 
    Return(leaseId)
}
```

Reply:
@serathius can you please comment on following questions
1.Any suggestion on how many total leases should be kept?
2.The 'Take' interface suggested above is marking a lease in-use. Does that mean multiple putwithlease requests should not use the same lease? How about we keep total pool of say 8 leases at a time and putwithlease will pick one randomly in-use or not?

Reply:
4 or 5, we can adjust them whenever.